soc/silabs/silabs_s2/Kconfig.defconfig

```diff
@@ -18,4 +18,10 @@ config SILABS_SLEEPTIMER_TIMER
 config CORTEX_M_SYSTICK
 	default n if SILABS_SLEEPTIMER_TIMER || GECKO_BURTC_TIMER
 
+config ZERO_LATENCY_IRQS
+	default y
+
+config ZERO_LATENCY_LEVELS
+	default 2
```

Comment:
Could you add some comments here as well, so that one doesn't need to trace back to which commit or PR this came from to understand the reasoning?